khelo/lib/ui/flow/home/home_screen.dart

```diff
@@ -81,12 +82,7 @@ class _HomeScreenState extends ConsumerState<HomeScreen>
     }
 
     if (state.matches.isNotEmpty) {
-      return ListView(
-        padding: context.mediaQueryPadding,
```

Comment:
@cp-mayank did we removed **context.mediaQueryPadding** completely? the difference in adding and not adding this padding would be seen in landscape orientation, please consider to add this padding if not added already.

Reply:
It's not necessary because we are not using a vertical scrollable widget. If we used a vertical scrollable widget, then the scroll container would have infinite height. That's why we need to give mediaQueryPadding to avoid hiding our items behind the system bar.

That Case we are not used scrollable Widget.

Reply:
in case of  single widget view will look like below, if you think below behaviour is correct then resolve the thread and react with cross sign on comment.
![Simulator Screenshot - iPhone 15 - 2024-05-20 at 17 44 50](https://github.com/canopas/khelo/assets/122426509/d6300c5c-83b6-4305-b414-05847d7c978f)